services/alertmanager/alertmanager.go

```diff
@@ -328,6 +332,97 @@ func (svc *Service) configAndReload(ctx context.Context, b []byte) error {
 	return nil
 }
 
+func convertTLSConfig(id string, tls *models.TLSConfig) promconfig.TLSConfig {
+	hashedIDBytes := sha256.Sum256([]byte(id))
+	hashedID := hex.EncodeToString(hashedIDBytes[:])
```

Comment:
Why do you hash ID?

Reply:
I cannot use the ID as a file name. Because it has entity type prefix 😞